comp/core/agenttelemetry/impl/config.go

```diff
@@ -171,37 +171,53 @@ var defaultProfiles = `
           aggregate_tags:
             - check_name
         - name: pymem.inuse
-        - name: pymem.alloc
+    schedule:
+      start_after: 30
+      iterations: 0
+      period: 900
+  - name: logs-and-metrics
+    metric:
+      exclude:
+        zero_metric: true
+      metrics:
+        - name: dogstatsd.udp_packets_bytes
+        - name: dogstatsd.uds_packets_bytes
+        - name: logs.bytes_missed
+        - name: logs.bytes_sent
         - name: logs.decoded
-        - name: logs.processed
-        - name: logs.sent
         - name: logs.dropped
-        - name: logs.sender_latency
-        - name: logs.bytes_sent
         - name: logs.encoded_bytes_sent
-        - name: logs.bytes_missed
+        - name: logs.processed
+        - name: logs.sent
         - name: point.sent
         - name: point.dropped
+        - name: transactions.input_count
+        - name: transactions.requeued
+        - name: transactions.retries
+    schedule:
+      start_after: 30
+      iterations: 0
+      period: 900
+  - name: database
+    metric:
+      exclude:
+        zero_metric: true
+      metrics:
         - name: oracle.activity_samples_count
         - name: oracle.activity_latency
         - name: oracle.statement_metrics
         - name: oracle.statement_plan_errors
-        - name: postgres.schema_tables_elapsed_ms
-        - name: postgres.schema_tables_count
+        - name: postgres.collect_activity_snapshot_ms
         - name: postgres.collect_relations_autodiscovery_ms
+        - name: postgres.collect_statement_samples_ms
+        - name: postgres.collect_statement_samples_count
         - name: postgres.collect_stat_autodiscovery_ms
-        - name: postgres.get_new_pg_stat_activity_ms
-        - name: postgres.get_new_pg_stat_activity_count
         - name: postgres.get_active_connections_ms
         - name: postgres.get_active_connections_count
-        - name: postgres.collect_activity_snapshot_ms
-        - name: postgres.collect_statement_samples_ms
-        - name: postgres.collect_statement_samples_count
-        - name: transactions.input_count
-        - name: transactions.requeued
-        - name: transactions.retries
-        - name: dogstatsd.udp_packets_bytes
-        - name: dogstatsd.uds_packets_bytes
+        - name: postgres.get_new_pg_stat_activity_count
+        - name: postgres.get_new_pg_stat_activity_ms
+        - name: postgres.schema_tables_elapsed_ms
+        - name: postgres.schema_tables_count
```

Comment:
These changes is mostly to be able to group metric in a more cohesive way, right?

Reply:
99% correct. It grew up too large to understand and use. I had to sacrifice a bit of complexity for that not to break efficiency with multiple profiles on the same schedule (to have a single batch payload) but essentially yes. 1% was removal of 2 unused metrics. I had to triple check the otherwise content had not been changed across previous code and across of Confluence page which list them.